content/messages/US/ppc_ni_nq.json

```
@@ -17,7 +17,10 @@
     "headline": [
         ["Buy now. Pay over time.", ["default", "xsmall", "small"]],
         [
-            ["No Interest if paid in full in 6 months", ["on purchases of {formattedMinAmount}+.", "weak"]],
+            [
+                "No Interest if paid in full in 6 months",
+                ["on PayPal purchases purchases of {formattedMinAmount}+.", "weak"]
```

Extra "purchases". You'll need to update the snapshots to match as well. 
```suggestion
                ["on PayPal purchases of {formattedMinAmount}+.", "weak"]
```